titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/ContainerHealth.java

```diff
@@ -0,0 +1,10 @@
+package com.netflix.titus.api.jobmanager.model.job;
+
+public enum ContainerHealth {
+    // Unset means we haven't gotten any signal yet about healthiness
```

Comment:
`Unset means we haven't gotten any signal yet about healthiness` -> `Unset means there has not been any signal yet about healthiness`